package.json

```diff
@@ -141,9 +141,11 @@
   "packageManager": "yarn@4.3.1",
   "resolutions": {
     "d3-color": "~3.1.0",
+    "express/path-to-regexp": "0.1.10",
```

Comment:
Can we add a `~` here?  I don't want to get stuck on this specific version if there are further fixes.